csharp/ql/lib/semmle/code/csharp/dataflow/internal/DataFlow.qll

```diff
@@ -233,17 +233,27 @@ module Make<ConfigSig Config> implements DataFlowSig {
   import Impl<C>
 }
 
+/** DEPRECATED: Use `Global` instead. */
+deprecated module Make<ConfigSig Config> implements DataFlowSig {
```

Comment:
Is this still not a breaking change since some of predicates in the DataFlowSig have been renamed?

Reply:
The old ones are still available.

Reply:
I guess it's breaking if you've written your own parameterised module that relies on `DataFlowSig` as input, but that signature was primarily meant as documentation.

Reply:
Fair enough. That seems unlikely!